docs/developer-guide/LayerFilter.md

```diff
@@ -128,11 +145,12 @@ Example:
 ```js
 {
     ogc: (filter::Object, options) => filter::String,
-    cql: (filter::Object, options) => filter::String
+    cql: (filter::Object, options) => filter::String,
+    geostyler: (filter::Object, options) => filter::[String]
 }
 ```
 
-`options` depends on the specific output format, but it can be used to pass additional parameters to the converter. For instance the `cql` convert has no options, but the `ogc` converter has an `options` object that can contain the `nsFilter` field, that is the srs of the geometry to be used in the filter. See the JSDoc of the `ogc` converter for more details.
+`options` depends on the specific output format, but it can be used to pass additional parameters to the converter. For instance the `cql` and instance `geostyler` converts have no options, but the `ogc` converter has an `options` object that can contain the `nsFilter` field, that is the srs of the geometry to be used in the filter. See the JSDoc of the `ogc` converter for more details.
```

Comment:
```suggestion
`options` depends on the specific output format, but it can be used to pass additional parameters to the converter. For instance the `cql` and `geostyler` converters have no options, but the `ogc` converter has an `options` object that can contain the `nsFilter` field, that is the srs of the geometry to be used in the filter. See the JSDoc of the `ogc` converter for more details.
```